app/src/main/res/values/styles.xml

```diff
@@ -1,6 +1,6 @@
 <resources>
   <!-- Base application theme. -->
-  <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
+  <style name="AppTheme" parent="Theme.MaterialComponents.Light.NoActionBar">
```

Comment:
the Material library provides many useful components. My tab layout doesn't work with the AppCompat theme. I checked other screens - seems fine with the material theme